charts/nr-k8s-otel-collector/values.yaml

```diff
@@ -76,6 +81,11 @@ deployment:
   containerSecurityContext: {}
   # -- Sets resources for the deployment.
   resources: {}
+  # -- Settings for deployment configmap
+  # @default -- See `values.yaml`
+  configMap:
+    # -- OpenTelemetry config for the deployment. If set, overrides default config and disables configuration parameters for the deployment.
+    config: {}
```

Comment:
Quick clarification. 

The value of config here will be replaced by the yaml from here

```
{{- if get .Values.deployment "configMap" | kindIs "map" -}}
    {{- if .Values.deployment.configMap.config -}}
        {{- toYaml .Values.deployment.configMap.config -}}
    {{- end -}}
{{- end -}}
```